app/code/Magento/Inventory/Model/Source.php

```diff
@@ -30,295 +30,298 @@ protected function _construct()
     /**
      * @inheritdoc
      */
-    public function getSourceCode()
+    public function getSourceCode(): ?string
     {
         return $this->getData(self::SOURCE_CODE);
     }
 
     /**
      * @inheritdoc
      */
-    public function setSourceCode($sourceCode)
+    public function setSourceCode(?string $sourceCode): void
     {
         $this->setData(self::SOURCE_CODE, $sourceCode);
     }
 
     /**
      * @inheritdoc
      */
-    public function getName()
+    public function getName(): ?string
     {
         return $this->getData(self::NAME);
     }
 
     /**
      * @inheritdoc
      */
-    public function setName($name)
+    public function setName(?string $name): void
     {
         $this->setData(self::NAME, $name);
     }
 
     /**
      * @inheritdoc
      */
-    public function getEmail()
+    public function getEmail(): ?string
     {
         return $this->getData(self::EMAIL);
     }
 
     /**
      * @inheritdoc
      */
-    public function setEmail($email)
+    public function setEmail(?string $email): void
     {
         $this->setData(self::EMAIL, $email);
     }
 
     /**
      * @inheritdoc
      */
-    public function getContactName()
+    public function getContactName(): ?string
     {
         return $this->getData(self::CONTACT_NAME);
     }
 
     /**
      * @inheritdoc
      */
-    public function setContactName($contactName)
+    public function setContactName(?string $contactName): void
     {
         $this->setData(self::CONTACT_NAME, $contactName);
     }
 
     /**
      * @inheritdoc
      */
-    public function isEnabled()
+    public function isEnabled(): ?bool
     {
-        return $this->getData(self::ENABLED);
+        return (bool)$this->getData(self::ENABLED);
```

Comment:
Need to check that initial values is not NULL
because null means not set, false means set false